### PR TITLE
Build Low-Touch Saas Guide page

### PR DESCRIPTION
**Pull request by mark-a-wilson**
This PR creates a new page for the Low-Touch Saas Guide (see #978), linked off of a new card on the Resources page.

Recognizing this is not the ideal implementation: content is hard-coded. I did my best to copy patterns already established where guide pages and accordions were used. But due to capacity and the fact that the content of the guide is unlikely to need to change in the near future, Steve asked me to just get it done this way in order to just get the content live.

**Note:** No tests were created for this new module as I didn't feel confident how to do that.

### CODE REVIEW

File: react-frontend/src/components/LowTouchSaas/collapsedMenus.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import Planning from './planning';
+import Selection from './selection';
+import PrivacyReview from './privacyReview';
+import SecurityReview from './securityReview';
+import LegalReview from './legalReview';
+import RiskManagementReview from './riskManagementReview';
+import ProcurementAndImplementation from './procurementAndImplementation';
+
+function CollapsedMenus({
+  planning,
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/LowTouchSaas/collapsedMenus.js:19]: These arguments don't seem to do anything

**Review by thegentlemanphysicist — CHANGES_REQUESTED** [review]
The file react-frontend/src/components/LowTouchSaas/collapsedMenuContent.js needs to be deleted since it's empty (I assume it was a temporary file) 
There are also some variables that are declared but never used.  They should be deleted.
Otherwise it looks good.

File: react-frontend/src/components/CoCos/coCoPage.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```
[the remark this answers is not in the document]
Comment (mark-a-wilson) [reply react-frontend/src/components/CoCos/coCoPage.js]: Oops! Thanks. Added back here:  https://github.com/bcgov/digital.gov.bc.ca/pull/980/commits/b060ebcf6f7dc622ba213af770c89c314d53f6f4

File: react-frontend/src/components/LowTouchSaas/collapsedMenus.js (hunk 1 above)
Comment (mark-a-wilson) [reply react-frontend/src/components/LowTouchSaas/collapsedMenus.js:18]: I honestly don't understand the code well enough here to know what could/should be removed. When I played around, I could not get the accordions to expand/collapse if I removed or changed code. 

The ideal would be _not_ to have the first accordion expanded by default (have them all closed by default) -- but it's not the end of the world.

Thank you for the review! I removed all the unused variables I could see here: https://github.com/bcgov/digital.gov.bc.ca/pull/980/commits/e587df903018476bd48e1d3edb53988b565b2a0e

Comment (thegentlemanphysicist) [reply react-frontend/src/components/LowTouchSaas/collapsedMenus.js]: I deleted them

**Review by thegentlemanphysicist — APPROVED** [review]
This looks good, deploys to dev properly

**Pull request merged**